modules/apps/portal-odata/portal-odata-impl/src/main/java/com/liferay/portal/odata/internal/filter/expression/ExpressionVisitorImpl.java

```diff
@@ -83,12 +92,23 @@ public Expression visitEnum(EdmEnumType edmEnumType, List<String> list) {
 
 	@Override
 	public Expression visitLambdaExpression(
-		String lambdaFunction, String lambdaVariable,
-		org.apache.olingo.server.api.uri.queryoption.expression.Expression
-			expression) {
-
-		throw new UnsupportedOperationException(
-			"Lambda expression: " + lambdaFunction);
+			String lambdaFunction, String lambdaVariable,
+			org.apache.olingo.server.api.uri.queryoption.expression.Expression
+				expression)
+		throws ExpressionVisitException, ODataApplicationException {
+
+		if (Objects.equals(
+				StringUtil.toUpperCase(lambdaFunction),
```

Comment:
It is not possible add Type to LambdaFunction, the same way than GE or EQ

Reply:
Unfortunately there is not a public constant defined for ANY.
Only constant found is private (in org.apache.olingo.server.core.debug.ExpressionJsonVisitor#ANY_NAME)